stdlib/public/Platform/TiocConstants.swift

```diff
@@ -0,0 +1,131 @@
+// TIOC constants, needed only on Darwin and FreeBSD
+
+#if os(OSX) || os(iOS) || os(tvOS) || os(watchOS) || os(FreeBSD)
+
+/// get modem control state
+public let TIOCMODG:UInt = 0x40047403
```

Comment:
Please use a get-only var instead `public var FOO: UInt { return 0xABCDE }`.

It causes fewer entry points to be emitted.